src/appleseed/renderer/modeling/aov/aovfactoryregistrar.cpp

```diff
@@ -89,8 +90,11 @@ void AOVFactoryRegistrar::reinitialize(const SearchPaths& search_paths)
     register_factory(auto_release_ptr<FactoryType>(new GlossyAOVFactory()));
     register_factory(auto_release_ptr<FactoryType>(new IndirectDiffuseAOVFactory()));
     register_factory(auto_release_ptr<FactoryType>(new IndirectGlossyAOVFactory()));
+    register_factory(auto_release_ptr<FactoryType>(new InvalidSampleAOVFactory()));
     register_factory(auto_release_ptr<FactoryType>(new NormalAOVFactory()));
+    register_factory(auto_release_ptr<FactoryType>(new PixelSampleAOVFactory()));
```

Comment:
I feel that this AOV name is ambiguous, I would suggest `PixelSampleCountAOV[Factory]`.